Generals/Code/GameEngine/Source/Common/System/GameMemory.cpp

```diff
@@ -3242,23 +3242,6 @@ void MemoryPoolFactory::debugMemoryReport(Int flags, Int startCheckpoint, Int en
 // GLOBAL FUNCTIONS
 //-----------------------------------------------------------------------------
 
-/*
-	This is a trick that is intended to force MSVC to link this file (and thus,
-	these definitions of new/delete) ahead of all others. (We do debug checking
-	to ensure that's the case)
-*/
-#if defined(_DEBUG)
-	#pragma comment(lib, "GameEngineDebug")
-#elif defined(_INTERNAL)
-	#pragma comment(lib, "GameEngineInternal")
-#else
-	#pragma comment(lib, "GameEngine")
-#endif
-
-#ifdef MEMORYPOOL_OVERRIDE_MALLOC
-	#pragma comment(linker, "/force:multiple")
-#endif
-
 static int theLinkTester = 0;
```

Comment:
Will theLinkTester work ok with the libs up there removed?

Reply:
It should do despite what the comments suggest about it altering the initialisation order. It wasn't the right way to force globals to be initialised before others anyhow. The /force:multiple is for allowing multiple definitions of the same symbol to be linked which is probably not something we actually want to happen and doesn't seem to be needed for any of the build configs we have.